docs/lang/articles/contribution/dev_install.md

```diff
@@ -341,22 +321,56 @@ You must install the Vulkan SDK in order to debug Taichi's Vulkan backend. To pr
 
 <TabItem value="linux">
 
-1. Clone the Taichi repo *recursively* and build[^1]:
+  ```shell
+  git clone --recursive https://github.com/taichi-dev/taichi
+
+  cd taichi
+
+  # Customize with your own needs
+  export TAICHI_CMAKE_ARGS="-DTI_WITH_VULKAN:BOOL=ON -DTI_WITH_CUDA:BOOL=ON"
+
+  # Uncomment if you want to use a different compiler
+  # export CC=/path/to/clang
+  # export CXX=/path/to/clang++
+
+  # export DEBUG=1 # Uncomment it if you wish to keep debug information.
+
+  # This would build a wheel file in `dist/` folder
+  ./build.py
+
+  # Alternatively, this would drop into a shell with complete build environment,
+  ./build.py --shell
+  # and then you could install Taichi in development mode
+  python3 setup.py develop
```

Comment:
I'd suggest put the alternative method in a separate section/code-block, so it's not gonna confuse the beginners.